cmd/combine/combine.go

```diff
@@ -133,42 +136,36 @@ func Combine(ctx context.Context, inputDir, outputDir string, force bool, opts .
 	return nil
 }
 
-func secretsToShares(lock cluster.Lock, secrets []tbls.PrivateKey) (map[int]tbls.PrivateKey, error) {
-	n := len(lock.Operators)
-
+func secretsToShares(lock cluster.Lock, secrets []tbls.PrivateKey, valIndex int) (map[int]tbls.PrivateKey, error) {
+	pubkMap := make(map[tbls.PublicKey]int)
 	resp := make(map[int]tbls.PrivateKey)
-	for idx, secret := range secrets {
-		pubkey, err := tbls.SecretToPublicKey(secret)
+
+	for idx := 0; idx < len(lock.Validators[valIndex].PubShares); idx++ {
```

Comment:
nit: lets use `peerIdx` to make explicit its 0-indexed. Getting confused between which indexes are 1 and 0 indexed if everything is just called `index`

Reply:
Lets double down and stay consistent to this terminology: `https://github.com/ObolNetwork/charon/blob/corver/clusterlegacy/cluster/definition.go#L25`

Reply:
The link is 404

Reply:
https://github.com/ObolNetwork/charon/blob/main/cluster/definition.go#L25